### PR TITLE
WIP: Add a Status Controller to Galley

### PR DESCRIPTION
**Pull request by ozevren**
This is not ready for check-in yet.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[X] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: install/kubernetes/helm/istio/charts/galley/templates/clusterrole.yaml [hunk 1]
```diff
@@ -14,18 +14,33 @@ rules:
...
   verbs: ["get", "list", "watch"]
+- apiGroups: ["rbac.istio.io"]
+  resources: ["*/status"]
+  verbs: ["update"]
 - apiGroups: ["security.istio.io"]
   resources: ["*"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["security.istio.io"]
+  resources: ["*/status"]
+  verbs: ["update"]
```
Comment (ayj) [inline install/kubernetes/helm/istio/charts/galley/templates/clusterrole.yaml]: FYI, these can be grouped to reduce duplication.

apiGroups: ["security.istio.io", "networking.istio.io", "rbac.istio.io"] # etc
resources: ["*/status"]
verbs: ["update"]

Comment (ayj) [reply install/kubernetes/helm/istio/charts/galley/templates/clusterrole.yaml]: At least I've verified that works for the get/list/watch verbs.

Comment (ozevren) [reply install/kubernetes/helm/istio/charts/galley/templates/clusterrole.yaml]: Thanks, I coalesced them together, both for watch etc and status.

**Pull request closed without merging**